src/formats/textbundle.ts

```diff
@@ -0,0 +1,77 @@
+import { normalizePath, Notice } from 'obsidian';
+import { parseFilePath, NodePickedFolder, NodePickedFile } from '../filesystem';
+import { FormatImporter } from '../format-importer';
+import { ProgressReporter } from '../main';
+
+export class TextbundleImporter extends FormatImporter {
+	init() {
+		this.addFileChooserSetting('Textbundle', ['textbundle']);
+		this.addOutputLocationSetting('Textbundle');
+	}
+
+	async import(progress: ProgressReporter): Promise<void> {
+		let { files } = this;
+		if (files.length === 0) {
+			new Notice('Please pick at least one file to import.');
+			return;
+		}
+
+		let folder = await this.getOutputFolder();
+		if (!folder) {
+			new Notice('Please select a location to export to.');
+			return;
+		}
+
+		let outputFolder = folder;
+
+		const attachmentsFolderPath = await this.createFolders(`${folder.path}/assets`);
+		const assetMatcher = /!\[\]\(assets\//g;
+
+		for (let file of files) {
```

Comment:
Maybe we should check the `info.json` first and make sure `type` is `net.daringfireball.markdown` if it's provided (looks like it's an optional field)?